packages/nocodb/src/lib/meta/api/userApi/ui/auth/resetPassword.ts

```diff
@@ -86,7 +86,7 @@ export default `<!DOCTYPE html>
             });
             this.success = true;
           } catch (e) {
-            alert('Some error occured')
+            alert('Failed to reset password: ' + e.response.data.msg)
```

Comment:
Exception here will only be in the form of `e.response.data` r8? Otherwise might cause issues

Reply:
`e.message` would return a general error message, i.e. `Request failed with status code 400` which is not a good message. We need to catch the error message thrown from API. That's why `e.response.data.msg` is used.

Reply:
What I meant was if there is an exception not from API call `e.response.data.msg` will error out. Probably won't happen